packages/main/src/components/Toolbar/index.tsx

```diff
@@ -297,10 +300,17 @@ const Toolbar = forwardRef<HTMLDivElement, ToolbarPropTypes>((props, ref) => {
   return (
     <CustomTag
       style={styleWithMinWidth}
-      className={clsx(toolbarClasses, overflowNeeded && classes.hasOverflow)}
+      className={clsx(
+        toolbarClasses,
+        overflowNeeded && classes.hasOverflow,
+        (currentTheme === 'sap_horizon' || currentTheme === 'sap_horizon_dark') && 'shadowOutline'
```

Comment:
I don't think we should read the current theme via JS here. Using the theme dependant CSS Variables would be the better fit in my opinion.